core/frontend/src/SubCategoriesCache.ts

```diff
@@ -111,13 +123,14 @@ export class SubCategoriesCache {
   /** Exposed strictly for tests.
    * @internal
    */
-  public add(categoryId: string, subCategoryId: string, appearance: SubCategoryAppearance) {
+  public add(categoryId: string, subCategoryId: string, appearance: SubCategoryAppearance, override: boolean) {
```

Comment:
What motivates the addition of the `override` parameter?

Reply:
There's a case shown in [ViewCreatorTest](https://github.com/iTwin/itwinjs-core/blob/d8f249a31d6d2325aa99e8c135d34aec6ac7c43f/full-stack-tests/core/src/frontend/standalone/ViewCreator3d.test.ts#L58), where the subcategories cache could be populated prior to loading the used spatial subcategories (aka prior to creating a default view). I wanted to keep the behavior of the existing test cases the same, and in case that behavior might be in real life